### PR TITLE
run e2e test by default

### PR DESCRIPTION
**Pull request by ofrobots**
@matthewloring PTAL.

### CODE REVIEW

**Comment by matthewloring** [comment]
@ofrobots Can you add a check that $encrypted_a8f6bb4bf8ae_key and $encrypted_a8f6bb4bf8ae_iv are defined before decryption in the before_install step?

**Comment by ofrobots** [comment]
@matthewloring PTAL.

File: bin/run-e2e.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -e
+
+cd test/e2e
+
+if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
```
Comment (matthewloring) [inline bin/run-e2e.sh]: Do we need to recheck the condition here?

**Comment by matthewloring** [comment]
LGTM (we should stop running against 2)

**Pull request merged**